app/views/reviews/_index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="rating">
+  <% @reviews.each do |review| %>
+    <%= review.rating %>
+    <%= review.body %>
+    <% if current_user %>
+      <li><%= link_to "Edit Review", edit_sitter_review_path(@sitter, review)  %></li>
+    <% end %>
+    <% if current_user %>
```

Comment:
Is there a reason why you needed two conditionals? Just test the existence of `current_user` once.